plugins/woocommerce/client/blocks/assets/js/data/validation/selectors.ts

```diff
@@ -36,3 +36,11 @@ export const getValidationErrorId = ( state: State, errorId: string ) => {
 export const hasValidationErrors = ( state: State ) => {
 	return Object.keys( state ).length > 0;
 };
+
+/**
+ * Gets all validation errors.
+ *
+ * @param { State } state The current state.
+ * @return { Record<string, any> } All validation errors.
```

Comment:
This is not correct, it would be `Record< string, FieldValidationStatus >`

Reply:
Updated. And also added an entry in the documentation